UI/DeviceConfigurationDlg.cs

```diff
@@ -126,7 +128,7 @@ internal DeviceConfigurationInfo ConfigurationInfo
             {
                 this.fitnessDevice.configInfo.ImportOnlyNew = this.chkImportOnlyNew.Checked;
                 this.fitnessDevice.configInfo.ImportSpeedDistanceTrack = this.chkImportDistance.Checked;
-                this.fitnessDevice.configInfo.DetectPausesFromSpeedTrack = this.chkDetectPausesFromSpeed.Checked;
+                //this.fitnessDevice.configInfo.DetectPauses = this.optDetectPausesFromSpeed.Checked;

```

Comment:
Add a comment here, noting that the value is read from defaults, updated at changes